src/test/end-to-end/tests/verified/email-subscriptions-test.ts

```diff
@@ -0,0 +1,290 @@
+        I.see('At least 1 subscription is needed.');
+
+        I.click('Add Subscriptions');
+        I.click('#subscription-choice-1');
+        I.click('Continue');
+        I.waitForText('What is the case reference number or case ID?');
+        I.fillField('#search-input', caseId);
+        I.click('Continue');
+        I.waitForText('Search result');
+        I.see('1 result successfully found');
+        I.see(caseId);
+        I.click('Continue');
+        I.click('Add another email Subscription');
+
+        I.waitForText('How do you want to add an email subscription?');
+        I.click('#subscription-choice-4');
+        I.click('Continue');
+        I.checkOption('//*[@id="' + locationId + '"]');
+        I.click('Continue');
+        I.waitForText('Confirm your email subscriptions');
+
+        I.click('Email subscriptions');
+        I.click('Add email subscription');
+        I.click('#subscription-choice-2');
+        I.click('Continue');
+        I.click('Continue');
+        I.waitForText('There is a problem');
+        I.see('There is nothing matching your criteria');
+        I.click('Add subscription by an alternative type');
+
+        I.click('#subscription-choice-1');
+        I.click('Continue');
+        I.click('Continue');
+        I.waitForText('There is a problem');
+        I.see('There is nothing matching your criteria');
+        I.click('Add subscription by an alternative type');
+
+        I.click('#subscription-choice-3');
+        I.click('Continue');
+        I.click('Continue');
+        I.waitForText('There is a problem');
+        I.see('Please enter a minimum of 3 characters');
+        I.fillField('#case-name', 'InvalidCaseName');
+        I.click('Continue');
+        I.waitForText('There is a problem');
+        I.see('There is nothing matching your criteria');
+        I.click('Add subscription by an alternative type');
+
+        I.click('#subscription-choice-4');
+        I.click('Continue');
+        I.click(locate('//input').withAttr({ value: 'Civil' }));
+        I.click(locate('//input').withAttr({ value: 'South East' }));
+        I.click('Apply filters');
+        I.checkOption('//*[@id="' + locationId + '"]');
+        I.click('Continue');
+        I.click('Confirm Subscriptions');
+        I.waitForText('Subscription(s) confirmed');
+        I.click('Email subscriptions');
+
+        I.click('#bulk-unsubscribe-button');
+        I.waitForText('Bulk unsubscribe');
+        I.click('#bulk-unsubscribe-button');
+        I.waitForText('There is a problem');
+        I.see('At least one subscription must be selected');
+        I.click(locate('//tr').withText(locationName).find('input').withAttr({ id: 'courtSubscription' }));
+        I.click('#bulk-unsubscribe-button');
+        I.waitForText('Are you sure you want to remove these subscriptions?');
+        I.click('#bulk-unsubscribe-choice');
+        I.click('Continue');
+        I.waitForText('Subscription(s) removed');
```

Comment:
Think we should also go back to the main email subscription screen at this stage and confirm there is no subscription left

Reply:
Fixed :)